renovate.json

```diff
@@ -7,7 +7,25 @@
   ],
   "automerge": false,
   "enabled": true,
+  "ignorePaths": [
+    "source/OptimaJet.DWKit.StarterApplication/package.json"
+  ],
   "packageRules": [
+    {
+      "packagePatterns": ["*.csproj"],
+      "ignoreDeps": [
+        "Microsoft.EntityFrameworkCore.InMemory",
```

Comment:
this isn't everything in an open PR now, but I figured either:
 - it'd be nice to be notified of certain updates (the DWKit nuget packages)
 - not part of dwkit (jwt auth, testing)

Reply:
I went through current open renovate PRs.  Either they were these, related to testing, or ones we added.

What about:

Microsoft.AspNetCore.Authentication.JwtBearer

Otherwise, looks good.